src/Sarif.Viewer.VisualStudio/CodeAnalysisResultManager.cs

```diff
@@ -239,24 +242,89 @@ public bool TryRebaselineCurrentSarifError(string uriBaseId, string originalFile
 
             string rebaselinedFile;
 
-            if (Uri.IsWellFormedUriString(originalFilename, UriKind.Absolute))
+            // File contents embedded in SARIF.
```

Comment:
>// [](start = 12, length = 2)

As written, this comment appears to apply to the whole `if`/`else` statement. Since it only applies to the `if` block, I'd place it in there:
```
if (FileDetails.ContainsKey(originalFilename))
{
    // File contents are embedded in the SARIF log file.
    rebaselinedFile = CreateFileFromContents(originalFilename);
}
else
{
    ...
}
```